### PR TITLE
Clean up BF downloads page for 5.3

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/1srf2EZR/22-download-page-cleanup
This PR updates the doc and javadoc zip links, fixes the icon which had the wrong file format on it and removes the decoupled formats-common component from the Bio-Formats download page for the 5.3.0 release.

Will need to be tested during an RC/milestone release.

### CODE REVIEW

**Comment by sbesson** [comment]
Closing this PR for 5.2.4 release. Can be reopened afterwards

File: bfgen.py [hunk 1]
```diff
@@ -49,16 +49,11 @@ def usage():
         ("formats-api.jar", "artifacts/formats-api.jar"),
         ("formats-bsd.jar", "artifacts/formats-bsd.jar"),
         ("formats-bsd-test.jar", "artifacts/formats-bsd-test.jar"),
```
Comment (sbesson) [inline bfgen.py]: This also needs to be removed for 5.3.0-m1

**Review by sbesson — CHANGES_REQUESTED** [review]

**Comment by hflynn** [comment]
@sbesson fix pushed

**Comment by sbesson** [comment]
It turns out that as part of https://github.com/openmicroscopy/bioformats/commit/840dc9b8b456548453e8d213366e16ea2b62d929 I also modified the name of the `loci_tools` JAR with is now `loci-tools`... Could you add an extra commit to fix this for m1? I'll revert this in a follow-up PR.

**Comment by hflynn** [comment]
@sbesson done

**Comment by sbesson** [comment]
Thanks. https://ci.openmicroscopy.org/job/BIOFORMATS-DEV-release-downloads/36/console now builds as expected.

**Review by sbesson — APPROVED** [review]

**Pull request merged**